app/src/androidTest/java/sdp/moneyrun/database/DatabaseProxyTest.java

```diff
@@ -39,38 +38,34 @@ public static void setPersistence(){
     }
     @Test
     public void getPlayerFromDatabase() {
-     //   FirebaseDatabase.getInstance().goOffline();
+       FirebaseDatabase.getInstance().goOffline();
```

Comment:
Why, is this useful in this context having an online Firebase Database would yield the same behavior no?